app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from typing import List
+
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    return sum([member.get_rating() for member in team])
```

Comment:
not need to create list 
```suggestion
    return sum(member.get_rating() for member in team)
```